app/models/service/containers.rb

```diff
@@ -16,7 +16,7 @@ def self.start_container!(wizard_state)
           container.send(:"#{taxonomy}=", wizard_state.preliminary.send(:"#{taxonomy}"))
         end
 
-        fail ActiveRecord::Rollback unless start_container(container)
+        fail ActiveRecord::Rollback unless pull_image(container) && start_container(container)
```

Comment:
What happens if the image was already pull? I assume self.pull_image will just return true?